net-im/psi/metadata.xml

```diff
@@ -9,17 +9,21 @@
 		<email>proxy-maint@gentoo.org</email>
 		<name>Proxy Maintainers</name>
 	</maintainer>
-	<longdescription>Psi is a very good XMPP client that uses Qt</longdescription>
+	<longdescription>Psi is a very good XMPP client that uses Qt and supporting wide range of XMPP
+		extensions. It's extremely configurable and extendable with plugins. It's supported on
+		almost every desktop system.
+	</longdescription>
```

Comment:
this needs a bit improving, how about this:
``
Psi is a multi-platform XMPP client based on Qt and supports a wide range of XMPP
extensions. It is extremely configurable and extendable with plugins.
``

I can change on merge if you ack.

Reply:
ack